.github/workflows/CypressImage.yaml

```diff
@@ -37,14 +37,14 @@ jobs:
           cp -r /home/runner/.gradle ./gradlecache
           cp -r /home/runner/.cache/Cypress ./cypresscache
       - name: Login to Github Packages
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Set up Docker Buildx
         id: buildx
-        uses: docker/setup-buildx-action@v2
+        uses: docker/setup-buildx-action@v3
```

Comment:
Soweit ich das sehen konnte gibt es setup-buildx-aciton nur in v2?!

Reply:
stimmt und es ist in der CI nicht aufgefallen weil die beiden container nicht Teil der CI sind